JonArnfred/week2/assignment3/README.md

```diff
@@ -0,0 +1,30 @@
+
+Some notes on decisions taken, there might be better alternative, would be happy to learn:
+
+- Using key as an id, since that makes access easy. Easier than storing messages and lists as arrays and looping to find.
```

Comment:
By building your data-model like this, it would be very hard/unreadable to do filters and in general get data out. You would have to iterate by keys/properties and that always(!) create more noise than necessary.
More complex code is always never better code in terms of readability and maintainability.
The code then doesn't scale good either if we want to add more types of lists and components in general